alacritty_terminal/src/term/mod.rs

```diff
@@ -222,9 +222,8 @@ impl<'a, C> RenderableCellsIter<'a, C> {
     ) -> RenderableCellsIter<'b, C> {
         let grid = &term.grid;
         let num_cols = grid.num_cols();
-        let num_lines = grid.num_lines();
 
-        let cursor_offset = grid.line_to_offset(term.cursor.point.line);
+        let cursor_offset = grid.num_lines().0 - term.cursor.point.line.0 - 1;
```

Comment:
I've inlined this method, since it was not used anywhere else anymore and even this one instance will disappear once the vimode PR is merged. So I don't think there's any need anymore to have it as separate method now.

We could keep it for now and just remove it in vimode, but I'd probably forget until this is merged and it should be fine the way it is.

Reply:
Makes sense.